README.md

```diff
@@ -332,11 +428,24 @@ prf = "prf" selector
 selector = "*" / 1*DIGIT
 ```
 
-`prf:*` represents all of the UCANs in the current proof scope. The witnesses in the `prf` field may be referenced by their index, starting at 0: `prf:0`
+`prf:*` represents all of the UCANs in the current proofs array. The witnesses for the current UCAN MAY be referenced by their index in the `"prf"` field. If selecting a particular witness (i.e. not the wildcard), then a zero-indexed MUST be used. The first UCAN would be selected by `prf:0`, the second by `prf:1`, and so on. By virtue of the indexing scheme, selections MUST be performed on the current UCAN only, and cannot recursively on nested witnesses.
 
-### 4.2.2 `prf` Actions
+Further selection of capabilities inside fo specific witnesses MUST NOT be a valid parsing of this URI. For instance, `prf:0:mailto` MUST NOT be a valid `prf` URI.
+
+### 4.3.2 `prf` Actions
```

Comment:
Re-reading this I'm wondering whether we can get away with only the `as:` *or* the `prf:` scheme, as they seem like they could both be used for e.g. device linking, although `prf:*` allows compacting UCAN delegations.

(I feel like I'm arguing against my past self)

I'm clarifying my thinking as I'm writing this.
`as:did:key:zMABC` must only be selecting all ambient authority of a certain DID, but *not* select the capabilities that the DID has in the particular UCAN chain,  or does it?

Reply:
You're right that these overlap a bit . The challenge with `prf` is that it doesn't let you reduce scope; it's really there to save on space so the thinking is that it should be as direct as possible and any use off of that would need to use some explicit form. So, if someone delegates all of their resources to me, I can't redelegate just all email addresses, for example.

Any thoughts?

Reply:
Oh I totally agree about disallowing `prf:0:mailto` and similar.

I was mostly thinking about the complexity vs. optimization gain trade-off.
But thinking about this again, it's not that complex at all.